components/core/src/os/ffi/windows.rs

```diff
@@ -8,13 +8,27 @@ pub trait OsStrExt3 {
 }
 
 pub trait OsStrExt2 {
+    #[allow(dead_code)]
```

Comment:
I think you can actually remove `OsStrExt2`. I really does look like dead code that is truly dead.